test/e2e/cypress/support/detail-panel-commands.js

```diff
@@ -0,0 +1,102 @@
+import { DATA_DETAIL, DETAIL_PANEL } from './selectors'
+
+Cypress.Commands.add('checkTermAndDefinition', (definitionlistNumber, listTerm, value) => {
+  if (value) {
+    cy.get(DETAIL_PANEL.definitionList)
+      .eq(definitionlistNumber)
+      .within(() => {
+        cy.get('[class*="DefinitionListTerm"]')
+          .contains(new RegExp(`^${listTerm}$`, 'g'))
+          .siblings(DETAIL_PANEL.definitionListDescription)
+          .contains(value)
+          .scrollIntoView()
+          .should('be.visible')
+      })
+  } else {
+    cy.get(DETAIL_PANEL.definitionList)
+      .eq(definitionlistNumber)
+      .within(() => {
+        cy.get('[class*="DefinitionListTerm"]')
+          .contains(new RegExp(`^${listTerm}$`, 'g'))
+          .scrollIntoView()
+          .should('be.visible')
+      })
+  }
+})
+
+Cypress.Commands.add('checkInfoBoxes', (arrayTerms) => {
+  cy.wrap(arrayTerms).each((term, i) => {
+    cy.get(DETAIL_PANEL.buttonInfo).eq(i).click()
+    cy.contains(`Uitleg over ${term}`).should('be.visible')
+    cy.get('[class*="DetailInfoBox"]').find(DETAIL_PANEL.buttonClose).eq(0).click()
+    cy.contains(`Uitleg over ${term}`).should('not.be.visible')
+  })
+})
+
+Cypress.Commands.add('checkListItems', (fixturePath) => {
+  cy.fixture(fixturePath).then((json) => {
+    // Check panel type and subject of the panel
+    cy.checkPanelHeader(fixturePath)
+
+    // Check for every itemlist all key-value pairs
+    const amountOfLists = Object.keys(json.definitionLists).length
```

Comment:
I'm not sure if this would work, as I'm writing this by heart, but this can be re-written like:
```suggestion
Object.entries(json.definitionLists).forEach(([keyA, valueA], indexA) => {
  cy.checkSubheader(fixturePath, indexA)
  Object.entries(valueA.items).forEach(([keyB, valueB], indexB) => {
    cy.checkTermAndDefinition(index, keyB, valueB)
  })
})
```

Reply:
Done!